openwisp_utils/settings.py

```diff
@@ -1,3 +1,4 @@
 from django.conf import settings
 
 EXTENDED_APPS = getattr(settings, 'EXTENDED_APPS', [])
+CUSTOM_ADMIN_SITE = getattr(settings, 'CUSTOM_ADMIN_SITE')
```

Comment:
move this to a new file in `admin_theme/settings.py`, since this is a setting of the admin_theme sub app and not the general openwisp-utils.

let's call this setting `OPENWISP_ADMIN_SITE_CLASS`.
Yes you will have to document this new feature.

You have to provide the default value of this setting, the default value has to be the path to the `OpenwispAdminSite` class